docker/Dockerfile

```diff
@@ -228,12 +236,8 @@ FROM dependencies-dev-platform-$TARGETARCH as dependencies-dev
 
 # Install node.js 18.x (LTS)
 # hadolint ignore=DL3008,DL4006
-RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash - && \
-    apt-get install --no-install-recommends -y nodejs && \
-    apt-get autoremove -y && \
-    apt-get clean all && \
-    rm -rf /var/lib/apt/lists/* && \
-    npm install --global markdownlint-cli@0.31.1
```

Comment:
Moved into `package.json`

Reply:
Since this isn't being installed globally it needs to be run with `npm exec markdownlint-cli` which may require a broader conversation. (FWIW this help us better deal with version differences)